packages/x-scheduler/src/primitives/use-event-calendar/tests/date.EventCalendarInstance.test.ts

```diff
@@ -0,0 +1,284 @@
+import { spy } from 'sinon';
+import { EventCalendarInstance } from '../EventCalendarInstance';
+import { getAdapter } from './../../utils/adapter/getAdapter';
+import { CalendarView } from '../../models';
+
+const DEFAULT_PARAMS = { events: [] };
+
+const adapter = getAdapter();
+
+describe('Date - EventCalendarInstance', () => {
+  describe('Method: goToToday', () => {
+    it('uncontrolled: sets visibleDate to startOfDay(adapter.date()) and calls onVisibleDateChange', () => {
+      const onVisibleDateChange = spy();
+      const { instance, store } = EventCalendarInstance.create(
+        { ...DEFAULT_PARAMS, onVisibleDateChange },
+        adapter,
+      );
+
+      const yesterday = adapter.addDays(adapter.startOfDay(adapter.date()), -1);
+      store.set('visibleDate', yesterday);
```

Comment:
We could pass `defaultVisibleDate: yesterday` to `EventCalendarInstance.create` I think